www/content/replicaset.md

```diff
@@ -0,0 +1,56 @@
+---
+title: ReplicaSet
+weight: 30
+menu: true
+---
+
+ReplicaSet
+
+Usually one does not create pods directly but create other resources like ReplicaSet's which creates and manages pods.
+(This because if node fails then those pods are lost.)
+
+Keeping pods healthy
+
+```shell
+
+workshop $ kubectl 
```

Comment:
If the "ReplicaSet" content is not done, maybe remove this file from this PR and create new PR when it's completed.